monkey/monkey_island/cc/services/edge/displayed_edge.py

```diff
@@ -0,0 +1,79 @@
+from copy import deepcopy
+
+from bson import ObjectId
+
+from monkey_island.cc.models.edge import Edge
+from monkey_island.cc.services.edge.edge import EdgeService
+
+__author__ = "itay.mizeretz"
+
+
+class DisplayedEdgeService:
+
+    @staticmethod
+    def get_displayed_edges_by_dst(dst_id, for_report=False):
+        edges = Edge.objects(dst_node_id=ObjectId(dst_id))
+        return [DisplayedEdgeService.edge_to_displayed_edge(edge, for_report) for edge in edges]
+
+    @staticmethod
+    def get_displayed_edge_by_id(edge_id, for_report=False):
+        edge = Edge.objects.get(id=edge_id)
+        displayed_edge = DisplayedEdgeService.edge_to_displayed_edge(edge, for_report)
+        return displayed_edge
+
+    @staticmethod
```

Comment:
Why is this a static method and not a regular method of Edge?

Reply:
I wanted to separate the data structure/schema of Edge document from methods, in order not to have a single huge file.  But you're right - we can do it without static method :) And I should make it into a boundary.